bin/detect

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env bash
 # bin/detect <build-dir>
 
-BUILD_DIR=$1
+build_dir="${1}"
 
-if [[ -n "$(find $BUILD_DIR -name *.war -type f)" ]]; then
-  echo "WAR"
-  exit 0
+if [ -n "${WAR_PATH}" ]
```

Comment:
```suggestion
if [[ -n "${WAR_PATH}" && -f "${build_dir}/${WAR_PATH}" ]] ; then
   [...]
elif [ -n "$( find "${build_dir}" -maxdepth 1 -name "*.war" -type f )" ] ; then
  [...]
fi
```

Reply:
What exactly do you mean by "old behavior"?
Normally it's kept when `WAR_PATH` is not set, yes, **EXCEPT** that default versions of both JRE and webapp-runner have been updated.
I guess this may have some consequences for some customers, but I don't measure the exact impact.

If necessary, it can be fixed by either:
- resetting the default versions in the buildpack
- or by instructing the customer to use the `JAVA_VERSION` and `JAVA_WEBAPP_RUNNER_VERSION` env vars.

Reply:
When I meant, running a war at the root of the project with the default versions ;-)

Reply:
Actually, java 1.8 is still the default on Heroku https://devcenter.heroku.com/articles/java-support and is still supported. I'm wondering if we should deviate here

Reply:
Thanks for the clarification. It's been successfully tested, yes :)

Reply:
OK, let's keep 1.8 then :)